package.json

```
@@ -71,7 +71,7 @@
     "cross-env": "^4.0.0",
     "eslint": "^2 || ^3 || ^4 || ^5 || ^6 || ^7.2.0",
     "eslint-import-resolver-node": "file:./resolvers/node",
-    "eslint-import-resolver-typescript": "^1.0.2 || ^1.1.1",
+    "eslint-import-resolver-typescript": "^1.0.2 || ^2.5.0",
```

eslint-import-resolver-typescript handles `@types` packages since [version 2.0.0](https://unpkg.com/browse/eslint-import-resolver-typescript@2.0.0/lib/index.js#L44).

---

i'd prefer to continue testing on v1.1.1
```suggestion
    "eslint-import-resolver-typescript": "^1.0.2 || ^1.1.1 || ^2.5.0",
```